query10.sql

```diff
@@ -4,4 +4,5 @@
 */
 
 -- Enter your SQL query here
-select ...
+select addressstreet, (the_geom <-> 'SRID=4326;POINT(-75.1923 39.9520)'::geometry) * 111139 as distance_to_Meyerson
```

Comment:
Should convert the geometry to some SRID other than 4326 (such as 2272 or 32129)